src/index.js

```diff
@@ -38,7 +38,7 @@ const envGraphQLValidatorNames = {
     'KnownFragmentNames',
     'NoUndefinedVariables',
     'NoUnusedFragments',
-    'ProvidedNonNullArguments',
+    'ProvidedRequiredArguments',
```

Comment:
this is likely to break the support for 0.12 and 0.13, correct? if so, perhaps we ought to do a runtime check to access the rules that the peer dependency `graphql` has. perhaps we should list _both_ of them here, and then filter based on what validators the installed version of `graphql` can support? this is exported on `require('graphql').specifiedRules`

would it be confusing to see both of these validators here?